### PR TITLE
gh-actions: add license check

### PR DESCRIPTION
**Pull request by acomodi**
Signed-off-by: Alessandro Comodi <acomodi@antmicro.com>

### CODE REVIEW

File: env.sh [hunk 1]
```diff
@@ -1,3 +1,10 @@
 #!/bin/bash
+# Copyright (C) 2021  The SymbiFlow Authors.
+#
+# Use of this source code is governed by a ISC-style
+# license that can be found in the LICENSE file or at
+# https://opensource.org/licenses/ISC
+#
+# SPDX-License-Identifier: ISC
 
 . `pwd`/env/conda/bin/activate fpga-interchange
```
Comment (mithro) [inline env.sh:10]: FYI - You should never use backticks, always use `$(xxx)`

**Review by mithro — COMMENTED** [review]
LGTM

File: env.sh (hunk 1 above)
Comment (acomodi) [reply env.sh:10]: All right, I'll fix that with a follow-up PR, thanks

**Comment by mithro** [comment]
FYI 
```
Error: :error file=tests/lut/lut.v,line=1,col=0:Missing SPDX-License-Identifier line in header
Error: :error file=tests/ram/ram.v,line=1,col=0:Missing SPDX-License-Identifier line in header
Error: :error file=tests/ram/ram_tb.v,line=1,col=0:Missing SPDX-License-Identifier line in header
Error: :error file=tests/wire/wire.v,line=1,col=0:Missing SPDX-License-Identifier line in header
Error: :error file=tests/wire/wire_tb.v,line=1,col=0:Missing SPDX-License-Identifier line in header
Error: :error file=utils/utils.v,line=1,col=0:Missing SPDX-License-Identifier line in header
```

**Pull request merged**